presqt/management/commands/validate_target_json.py

```diff
@@ -0,0 +1,41 @@
+import json # pragma: no cover
+
+from django.core.management import BaseCommand # pragma: no cover
+
+
+from presqt.json_schemas.schema_handlers import schema_validator # pragma: no cover
+
+
+
+class Command(BaseCommand): # pragma: no cover
+    def handle(self, *args, **options):
+        """
+        Verify that the Target JSON file is valid against our JSON Schema
+        """
+        validation = schema_validator(
+            'presqt/targets.json',
+            'presqt/json_schemas/target_schema.json'
+        )
+
+        # If JSON Schema validation has failed
+        if validation is not True:
+            print(validation)
+            print("Target JSON Schema Validation Failed!")
```

Comment:
Maybe a bit more explanation to the end user would be good? I imagine they might lost on what the "Target JSON Schema" was.